provisioner/powershell/provisioner.go

```diff
@@ -380,9 +380,14 @@ func (p *Provisioner) createFlattenedEnvVars(elevated bool) (flattened string) {
 	envVars["PACKER_BUILD_NAME"] = p.config.PackerBuildName
 	envVars["PACKER_BUILDER_TYPE"] = p.config.PackerBuilderType
 
+	// expose PACKER_HTTP_ADDR
 	httpAddr := common.GetHTTPAddr()
+	httpAddrs := strings.Split(httpAddr, ":")
+	httpIP, httpPort := httpAddrs[0], httpAddrs[1]
```

Comment:
Question here: is it guaranteed here that `httpAddrs` will always contain 2 fields, this might panic.

Also; the common pkg has `SetHTTPIP` `SetHTTPPort` may be we could give it getters to avoid this; what do you think ?

Reply:
Yes it should always have to fields, but also yes, getters makes more sense since I'm implementing the same query in several places.